File: src/Farmer.java
public class Farmer {
    public String getPhoneNumber;
    private String name;
    private String phoneNumber;
    private String farmAddress;
    private String utilityBillPath;
//        path to utility bill image

//    Constructors
    public Farmer(String name, String phoneNumber, String farmAddress, String utilityBillPath) {
        this.name = name;
        this.phoneNumber = phoneNumber;
        this.farmAddress = farmAddress;
        this.utilityBillPath = utilityBillPath;
    }
//    GETTERS AND SETTERS FOR ALL ATTRIBUTES
        public String getName() {
            return name;
        }
    public void setName(String name){
        this.name = name;
        }

        public String getPhoneNumber () {
            return phoneNumber;
        }
    public void setPhoneNumber(String phoneNumber) {
        this.phoneNumber = phoneNumber;
    }

        public String getFarmAddress() {
            return farmAddress;
        }
    public void setFarmAddress(String farmAddress) {
        this.farmAddress = farmAddress;
    }

        public String getUtilityBillPath() {
        return utilityBillPath;
        }
    public void setUtilityBillPath(String utilityBillPath) {
        this.utilityBillPath = utilityBillPath;
    }
}

File: Backend/models/farmer.py
class Farmer:
    def __init__(self, name, phone_number, farm_address, utility_bill_path):
        self.name = name
        self.phone_number = phone_number
        self.farm_address = farm_address
        self.utility_bill_path = utility_bill_path

    def get_name(self):
        return self.name

    def set_name(self, name):
        self.name = name

    def get_phone_number(self):
        return self.phone_number

    def set_phone_number(self, phone_number):
        self.phone_number = phone_number

    def get_farm_address(self):
        return self.farm_address

    def set_farm_address(self, farm_address):
        self.farm_address = farm_address

    def get_utility_bill_path(self):
        return self.utility_bill_path

    def set_utility_bill_path(self, utility_bill_path):
        self.utility_bill_path = utility_bill_path
